packages/api/lambdas/in-region-s3-policy.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const got = require('got');
+const policyTemplate = require('./bucket-policy-template.json');
+
+const defaultRegion = 'us-east-1';
```

Comment:
Could you leave a comment here that this is hard coded because it is a prototype